.vscode/README.md

```diff
@@ -0,0 +1,21 @@
+VSCode Setup
+=======
+
+## Introduction
+Java development is not supported out of the box and several plugins needs to be installed and configured. Once this setup is complete VSCode offers a good development environment. It will enable intellisense, spell checking, code formatting and checksyles. The versions of plugins defined were the most up to date and tested versions at the time of writing.
+
+This guide assumes that VSCode is already installed.
+
+### Extension
+The file ```extensions.json``` in this repository shows which extensions have been used in this setup. They will need installing via the Extension tab. 
+
+The installed extensions will be installed at ```~/.vscode-server/extensions/``` along with a copy of ```extensions.json```
```

Comment:
I think this would read better as `The extensions will be installed at ....`

Reply:
I have changed the wording